### PR TITLE
Frontenac integration

### PR DESCRIPTION
**Pull request by mprati**
Note that the functionality of  "Octave/octave_setup.py" and "Octave/octave_setup_cc.py" have been integrated into "cPRONTO/setup_oppni.py" and should no longer need to be run independently.

### CODE REVIEW

**Comment by raamana** [comment]
Awesome! Let's review this in person, going through the changes one by one, Mark.

File: Octave/octave_setup.py [hunk 1]
```diff
@@ -1,43 +1,37 @@
...
+        sys.exit()
 
-    # Install the packages from within octave, in order
-    # If you have SUDO, and/or you dont have lib-dev Octave
     try:
-         subprocess.call("apt-get install -y octave-io octave-control octave-struct octave-statistics octave-signal octave-optim")
-         print "Installed Octave Packages with apt-get"
-    except:
-        # If you dont have SUDO, but you do have lib-dev Octave
+        # Install the packages from within octave, in order
```
Comment (raamana) [inline Octave/octave_setup.py:20]: Remove this deprecated file

File: Octave/octave_setup_cc.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Mark Prati
+# Modified to set up an Octave environment for running OPPNI on Compute Canada (Cedar)
+#
+import os
+import subprocess
+
+def main():
+    """To be called when using octave for the first time"""
+
+    # Load the CC module
```
Comment (raamana) [inline Octave/octave_setup_cc.py:11]: delete deprecated file

File: Octave/mass_verification.py [hunk 3]
```diff
@@ -22,25 +44,52 @@ def main():
...
+                continue
+
             # Get call ready
+            print("matlabfile = ",matlabfile)
+            print("octavefile = ",octavefile)  
+          
             savefile = pjoin(version_check_location,option,"MvsO_results.txt")
-            print(savefile)
+            print("Results will be saved to: ", savefile)
+
```
Comment (raamana) [inline Octave/mass_verification.py:128]: run status checks for both Matlab and Octave processing folders, via main wrapper status check, before launching version check

File: scripts_matlab/filtfilt_octave.m [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  endif
+  si(1) = [];
+
+  ## This loop generally executes tens of thousands of times.
+  ## For optimization all repetitivly redundent calculation should be moved ouside the loop body.
+  ## Mods by Mark Prati
+  ###########################
+  sx2 = size(x,2);
+  lrefl1 = lrefl + 1;
+  lxlrefl = lx + lrefl;
```
Comment (raamana) [inline scripts_matlab/filtfilt_octave.m:87]: Have a copy of the previous unmodified code in the commented form, for future reference!

**Review by raamana — COMMENTED** [review]
Remove deprecate files:
```
Octave/octave_setup.py 
Octave/octave_setup_cc.py
```

**Comment by raamana** [comment]
In `Octave/mass_verification.py`, line 39:

run status checks for both Matlab and Octave processing folders, via main wrapper status check, before launching version check

**Comment by raamana** [comment]
within `scripts_matlab/filtfilt_octave.m`

Have a copy of the previous unmodified code in the commented form, for future reference!

File: Octave/octave_setup_cc.py (hunk 2 above)
Comment (mprati) [reply Octave/octave_setup_cc.py:11]: done

**Comment by mprati** [comment]
> within `scripts_matlab/filtfilt_octave.m`
> 
> Have a copy of the previous unmodified code in the commented form, for future reference!

filtfilt_octave.m has been specifically added to support Octave and is not in the execution path when matlab is selected.

**Pull request closed without merging**